docker-compose.dev.yml

```diff
@@ -38,6 +38,7 @@ services:
       - SHELLHUB_ENTERPRISE=${SHELLHUB_ENTERPRISE}
       - SHELLHUB_CLOUD=${SHELLHUB_CLOUD}
       - SHELLHUB_ENV=${SHELLHUB_ENV}
+      - SHELLHUB_LOG_LEVEL=${SHELLHUB_LOG_LEVEL}
```

Comment:
I believe `dev` extends the original docker-compose, so I *think* this block could remove this. @gustavosbarreto, am I right?